### ISSUE TITLE
Missing ALAG1 and D1 parameters and RATE-column

### ISSUE THREAD

**Issue by stellabelin** [pharmpy/pharmpy#578]
ALAG1, D1 and an added RATE-column are missing when setting to ZO elimination, adding lagtime and ZO absorption for [this](https://github.com/pharmpy/pharmpy/blob/master/tests/testdata/nonmem/models/mox2.mod) model. If `model_code` is called after all transformations the code looks okay. The following code can be used to reproduce the error :
```
set_zero_order_elimination(model)
model.model_code
add_lag_time(model)
set_zero_order_absorption(model)
print(model.model_code)
```
This produces the NONMEM code:
```
$INPUT ID VISI DAT1 DGRP DOSE FLAG ONO XIME DVO NEUY SCR AGE SEX NYHA WT COMP IACE DIG DIU NUMB TAD TIME VID CRCL AMT SS II VID1 CMT=DROP CONO DV EVID OVID SHR SHR2
...
$PK
MAT = THETA(5)
MDT = THETA(4)
KM = THETA(3)
CLMM = THETA(2)*EXP(ETA(1))
VC = THETA(1)*EXP(ETA(2))
V = VC
```

**Comment by stellabelin** [comment]
Also occurs with MM elimination

**Comment by stellabelin** [comment]
Fixed in 951d65239c788240632bafb831c01ec03e482c51 and 673a576d969c61a65705e4fd24054d91db1dbd1a
